README.md

```diff
@@ -32,27 +32,47 @@
 
 *   If installing manually, you will need Python3 (3.8 or higher), [gffutils](https://pythonhosted.org/gffutils/installation.html), [pysam](https://pysam.readthedocs.io/en/latest/index.html), [pybedtools](https://daler.github.io/pybedtools/), [biopython](https://biopython.org/) and some other common Python libraries to be installed. See `requirements.txt` for details. You will also need to have [minimap2](https://github.com/lh3/minimap2) and [samtools](http://www.htslib.org/download/) to be in your `$PATH` variable.
   
+*   Verify installation
+
+        isoquant.py --test
+
 *   To run IsoQuant on raw FASTQ/FASTA files use the following command
 
-        isoquant.py --reference /PATH/TO/reference_genome.fasta 
-        --genedb /PATH/TO/gene_annotation.gtf 
-        --fastq /PATH/TO/sample1.fastq.gz /PATH/TO/sample2.fastq.gz 
+        isoquant.py --reference /PATH/TO/reference_genome.fasta \
+        --genedb /PATH/TO/gene_annotation.gtf \
+        --fastq /PATH/TO/sample1.fastq.gz /PATH/TO/sample2.fastq.gz \
         --data_type (assembly|pacbio_ccs|nanopore) -o OUTPUT_FOLDER
 
+    For example, using the toy data provided within this repository,
+
+        mkdir toy_data_fastq_assembly
```

Comment:
you don't need to create output folder beforehand, it will be created automatically